pkg/cli/node.go

```diff
@@ -0,0 +1,115 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/QMSTR/qmstr/pkg/builder"
+	"github.com/QMSTR/qmstr/pkg/common"
+	"github.com/QMSTR/qmstr/pkg/gnubuilder"
+	"github.com/QMSTR/qmstr/pkg/qmstr/service"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var (
```

Comment:
I see no reason for global vars. For "target" i explain below. "input" & "mode" are not used anywhere besides createFileNodes function. Why are they global? Possibly a mistake.